sky/engine/bindings/flutter_dart_state.cc

```diff
@@ -34,7 +33,7 @@ FlutterDartState::~FlutterDartState() {
 }
 
 FlutterDartState* FlutterDartState::CreateForChildIsolate() {
-  return new FlutterDartState(isolate_client_, url_);
+  return new FlutterDartState(isolate_client_);
```

Comment:
The pointer ends up being stored in side Dart_Isolate and manually deleted in the isolate shutdown callback.  I could make this return a unique_ptr, but we'd have to release it right away to put it in the isolate.